python/paddle/distributed/launch.py

```diff
@@ -139,6 +139,7 @@ def start_procs(args):
 
     current_node_ip = args.node_ip
     node_ips = [x.strip() for x in args.cluster_node_ips.split(',')]
+    node_num = len(node_ips)
     node_id = node_ips.index(current_node_ip)
     num_nodes = len(node_ips)
```

Comment:
variable ```node_num```  equals to ```num_nodes```. maybe we can change all ```num_nodes``` to ```node_num``` and rm ```num_nodes```.

Reply:
Thanks! Done!